drivers/flash/Kconfig.stm32

```diff
@@ -84,6 +84,14 @@ config FLASH_STM32_BLOCK_REGISTERS
 	  registers improves system security, because flash content (or
 	  protection settings) can't be changed even when exploit was found.
 
+config FLASH_STM32_OPTION_BYTES
+	bool "Extended operation for option bytes access"
+	select FLASH_HAS_EX_OP
+	default n
+	help
+	  Enables flash extended operations that can be used to write and write
+	  STM32 option bytes.
```

Comment:
```suggestion
	  Enables flash extended operations that can be used to write and read
	  STM32 option bytes.
```

Reply:
see, if my github account got compromised you'll be able to tell immediately as soon as you see a PR of me without at least one typo